doc/api.rst

```diff
@@ -2,12 +2,40 @@
 API
 ###
 
+This is the reference for classes (``CamelCase`` names) and functions
+(``underscore_case`` names) of MNE-Connectivity, grouped thematically by analysis
+stage. MNE-Connectivity also provides connectivity data structure
+containers for different types of connectivity data, which are
+described below.
+
 :py:mod:`mne_connectivity`:
 
 .. automodule:: mne_connectivity
    :no-members:
    :no-inherited-members:
```

Comment:
Looking at the rendered Documentation, these lines left me a bit puzzled. It seems to show up as:
```
mne_connectivity:

Connectivity MEG, EEG, iEEG data processing.
```

and `mne_connectivity_  just links to the next line. (Looking at the last commit of July 2nd)

Reply:
True, I think I copied these lines from MNE-realtime/Python. I took these out though since there was no real purpose.